libbeat/esclientleg/api_test.go

```diff
@@ -172,8 +174,8 @@ func TestReadSearchResult_invalid(t *testing.T) {
 	assert.Error(t, err)
 }
 
-func newTestClient(url string) *Client {
-	client, err := NewClient(ClientSettings{
+func newTestClient(url string) *elasticsearch.Client {
```

Comment:
TODO: Looks like this test needs to be moved back to the `elasticsearch` package, at least for now.